src/metalnx-web/src/main/java/com/emc/metalnx/controller/LoginController.java

```diff
@@ -143,12 +146,16 @@ public ModelAndView loginErrorHandler(final Exception e) {
 		 * see if I'm really logged in as a hacky way of getting around this bug
 		 * https://github.com/irods-contrib/metalnx-web/issues/162
```

Comment:
this issue is now closed... please consider this comment and remove/update as necessary

Reply:
definitely does not have to be part of this PR.... just caught my eye

Reply:
Don't we want a comment that explains why the additional code is needed?  I don't see how #162 being closed removes the utility of the comment.

Reply:
I changed the comment to add clarity.  Please take a look.